markdown/ch3.markdown

```diff
@@ -8,7 +8,7 @@
 > — William Blake, *The Tyger*
 
 You probably know that digital images are made up of *pixels*. These are
-the light signal *sampled on a regular grid*. When computing
+*samples* of the light signal *on a regular grid*. When computing
```

Comment:
Excellent.  I mean, the way we sample is, of course, to integrate over an almost-square CCD element, but... let's not get into the weeds :seedling: